projects/addon-mobile/directives/dropdown-mobile/dropdown-mobile.template.html

```diff
@@ -3,7 +3,7 @@
     waIntersectionObserver
     waIntersectionThreshold="1"
     class="t-filler"
-    (touchstart.prevent)="close()"
+    (pointerdown.stop)="close()"
```

Comment:
Otherwise, it will also prevent `mouseout` event
https://github.com/taiga-family/taiga-ui/blob/80b3257f0c34fcb3c64483f6678bc88c4ae702a5/projects/core/directives/dropdown/dropdown-hover.directive.ts#L35-L42

It means that all subsequent clicks will be prevented.

https://github.com/taiga-family/taiga-ui/blob/80b3257f0c34fcb3c64483f6678bc88c4ae702a5/projects/core/directives/dropdown/dropdown-hover.directive.ts#L62-L64